mapboxgl/templates/vector_choropleth.html

```diff
@@ -0,0 +1,104 @@
+{% extends "choropleth.html" %}
+    
+    {% block choropleth %}
+
+    var joinData = {{ joinData }};
```

Comment:
We don't want to store the raw data, since we re-map this into an object for the tooltip and array for the style below.  Scope this as a function and import the variable in the function scope using `let joinData = {{ joinData }}`.